docs/canvas/canvas-getting-started.asciidoc

```diff
@@ -2,31 +2,145 @@
 [[canvas-getting-started]]
 == Getting started with Canvas
 
-Your best bet to getting started with Canvas is to check out one 
-(or all) of the sample data sets that ship with {kib}. 
-
-. Click the {kib} logo in the upper left hand corner of your browser to navigate 
-to the {kib} home page.
-. Click *Load a data set and a Kibana dashboard*. This also loads a 
-Canvas workpad to go with the data set.
-. Pick a data set. Let’s go with the eCommerce one for this example, but while you're 
-here, feel free to click *Add* on all of the available sample data sets.
-. Click the Canvas icon in the left hand navigation menu.
-. Find the *[eCommerce] Revenue Tracking* workpad and click on its name to open it.
+To get up and running with Canvas, use the following tutorial where you'll create a display for monitoring sales at an eCommerce store.
```

Comment:
Do we need to add a pre - requisite section here- and talk a little bit about Elasticsearch - needed for storing and indexing the document ?

Reply:
Good question! I'm not sure. Are there configuration settings that users need to change in order to correctly store and index their workpad?

Reply:
A link for installing the sample data set is right below this statement.